tfe/provider_test.go

```diff
@@ -369,12 +369,20 @@ func testAccGithubPreCheck(t *testing.T) {
 	}
 }
 
+func testAccGHAInstallationPreCheck(t *testing.T) {
+	testAccPreCheck(t)
```

Comment:
Remove duplicate `testAccPreCheck(t)` since test case `TestAccTFEWorkspace_updateGitHubAppRepo` invoke this method as well

Reply:
removed all the duplicate calls from the test functions.